src/sentry/utils/numbers.py

```diff
@@ -0,0 +1,26 @@
+ALPHABET = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ'
```

Comment:
We might want to drop 0, 1, I, and O since they can be confusing visually and use [Base32](https://en.wikipedia.org/wiki/Base32) instead:

![screenshot 2016-03-07 18 26 23](https://cloud.githubusercontent.com/assets/65315/13590104/1d0d6346-e492-11e5-90e4-4a3f40f5c557.png)